src/css/Splash.module.css

```diff
@@ -0,0 +1,42 @@
+.splash {
+  margin: 0;
+  color: white;
+  background-image: url(../assets/laptop.jpg);
+  background-size: cover;
+  height: 100vh;
+}
+
+.blur {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  backdrop-filter: blur(10px) brightness(0.1);
```

Comment:
Please consider to add -webkit-backdrop-filter to support safari also and backdrop-filter is not supported by firefox. (ps: I personally use firefox. I don't see effect on it)

Reply:
Done

Reply:
NP